cmd/kubelet/app/options/options.go

```diff
@@ -114,6 +112,7 @@ func NewKubeletServer() *KubeletServer {
 			RktPath:                        "",
 			RktStage1Image:                 "",
 			RootDirectory:                  defaultRootDir,
+			RuntimeContainer:               "/docker-daemon",
```

Comment:
All the container defaults are wrong for systemd. They have to be set to `""` as I mentioned above.